juju/juju
# Do not allow restore-backup in HA config.

## Description of change

Do not allow restore-backup in HA config.

## QA steps

1. juju bootstrap ; juju enable-ha
2. juju switch controller
3. juju create-backup --no-download
4. juju restore-backup --id <id from step 3>  <-- should fail
5. juju remove-machine 1 2  <- wait for machines to be removed
4. juju restore-backup --id <id from step 3>  <-- should succeed

## Documentation changes

Yes

## Bug reference

N/A
